### DISCUSSION TITLE
Does transferring data cost more energy, than storing it nearby?

### DISCUSSION THREAD

**Discussion by therealmaxmoon** [alexnathanson/solar-protocol#90] [General]
I am pretty fascinated by the project and I read almost everything on http://solarprotocol.net/ and I am a fan of the lowtech magazine, too.

But as always with sustainable projects, there are doubt's after analyzing it. 

On the website I read:

> We know that the computational cycles required to generate our imagery are powered by solar energy rather than potentially powered as fossil energy in the browser. This design approach also minimizes the energy consumption for the user.

The solar protocol gets the server, which currently gets the best sustainable energy. In many cases those are countries near the equator. As I surfed on http://solarprotocol.net/ I was on a server in Afrika. 

But Afrika is really far away and the files must be provided over several servers. Those servers are most likely powered by fossil energy and the longer the route, the more energy is needed to provide the website.

**My question is:** Is it really sustainable to provide the website from a country were the most energy is produced if the transportation costs more energy compared of connecting to a server nearby, even if it's night and the server runs on battery (which was charged with solar power before)?

Sorry to use the git repo for asking questions, but I couldn't find a channel on reddit, matrix, mastodon or any other platform.

**Comment by samuk** [comment]
This is a good question. If it scaled to the point you had several online servers at any given time you could use a CDN to serve from the one geographically closest to you.

If I'm reading [this  study](https://hal.science/hal-03196527/document) correctly we could work with a figure of ~1g CO2 equivalent per GB data per 750Km sent

**Reply by therealmaxmoon** [reply]
~1g CO2 equivalent per GB data per 750Km sounds really not much, so it sounds like it really doesn't matter where the server is standing, if it's about the produced CO2e, right?

P.S.: I wanted to download the study and put it to my pile of unread pdfs and it's already there 😅
